src/CreatorTokenSwapRouter.sol

```diff
@@ -18,6 +21,10 @@ contract CreatorTokenSwapRouter {
   // WETH -> USDC, order is switched because of V3_SWAP_EXACT_OUT
   bytes private path;
```

Comment:
We should have `@notice` natspec comments for each of these constants and storage vars as well

Reply:
added here: [e159a53](https://github.com/showtime-xyz/creator-tokens/pull/46/commits/e159a53031291a7ef77626ca2d681aec08053372)